include/dm.h

```diff
@@ -7,7 +7,13 @@
 #ifndef DM_H_
 #define DM_H_
 
-/**@file
+#include <zephyr/kernel.h>
+#include <stdlib.h>
```

Comment:
Please move it above `<zephyr/kernel.h>` and add a blank line after. It will look more readable and nice

Reply:
ACK